From b52307797b91d7448a72874e4da7bed30c27a4f5 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@lantiq.com>
Date: Tue, 21 Jul 2015 22:29:14 +0200
Subject: [PATCH] soft-sensor-manager: libSSMCore.so: add missing depending
 library

libSSMCore.so also depends on libdl.so, this patch adds these
dependencies to the build.

Without this the dynamic loader will not automatically load these
libraries and it could result in unresolved dependencies at runtime.

Change-Id: I76faff46fe96caef75fb6673c7e83d62b15b8489
Signed-off-by: Hauke Mehrtens <hauke.mehrtens@lantiq.com>
Reviewed-on: https://gerrit.iotivity.org/gerrit/1796
Tested-by: jenkins-iotivity <jenkins-iotivity@opendaylight.org>
Reviewed-by: Uze Choi <uzchoi@samsung.com>
---
 service/soft-sensor-manager/SConscript |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/service/soft-sensor-manager/SConscript
+++ b/service/soft-sensor-manager/SConscript
@@ -157,7 +157,7 @@ static_libssmcore = ssmcore_env.StaticLi
 
 ssmcore_env.AppendUnique(LIBPATH = [env.get('BUILD_DIR')])
 ssmcore_env.AppendUnique(LIBS = ['oc', 'octbstack', 'oc_logger',
-            'connectivity_abstraction', 'coap'])
+            'connectivity_abstraction', 'coap', 'dl'])
 
 shared_libssmcore = ssmcore_env.SharedLibrary(
 		target = 'SSMCore',
